src/styles/style.css

```diff
@@ -0,0 +1,15 @@
+@import url(blocks/stars.css);
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: #fff;
+  height: 96px;
+  width: 96px;
+  flex-shrink: 0;
+}
```

Comment:
- In your case `.container` is a BEM block:
Each BEM block has its own separate file
- Also, you actually don't need it at all for this task